x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/store/input/FrozenIndexInput.java

```diff
@@ -209,55 +217,6 @@ private int readCacheFile(
         return bytesRead;
     }
 
-    private void writeCacheFile(
-        final SharedBytes.IO fc,
-        final InputStream input,
-        final long fileChannelPos,
-        final long relativePos,
-        final long length,
-        final LongConsumer progressUpdater,
-        final long startTimeNanos
-    ) throws IOException {
-        assert ThreadPool.assertCurrentThreadPool(SearchableSnapshots.CACHE_FETCH_ASYNC_THREAD_POOL_NAME);
-        logger.trace(
-            "{}: writing channel {} pos {} length {} (details: {})",
-            fileInfo.physicalName(),
-            fileChannelPos,
-            relativePos,
-            length,
-            cacheFile
-        );
-        final long end = relativePos + length;
-        logger.trace("writing range [{}-{}] to cache file [{}]", relativePos, end, cacheFile);
```

Comment:
I removed this log line when copying, it adds no information that isn't also found in the log line above.